starter_code/src/movies.js

```diff
@@ -1,15 +1,148 @@
 /* eslint no-restricted-globals: 'off' */
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
+function orderByYear(movies) {
+   
+    let newMovies = [...movies];
+    if ( movies.length == 0  ){
+       return newMovies ;
+    }
+    
+    newMovies.sort(function(moviea, movieb) { 
+        return moviea.year - movieb.year  ;
+    });
+    
+    newMovies.sort((a, b) => (a.year > b.year) ? 1 : (a.year === b.year) ? ((a.title > b.title) ? 1 : -1) : -1 )
```

Comment:
Here you could blend in your sort functions.
```

function orderByYear(movies) {
   
    let newMovies = [...movies];
    if ( movies.length == 0  ){
       return newMovies ;
    }
    
    newMovies.sort(function(moviea, movieb) { 
        return moviea.year === movieb.year ? moviea.title.localeCompare(movieb.title) : moviea.year - movieb.year
        // More detailed : 
        // if(moviea.year === movieb.year){
        //     return moviea.title.localeCompare(movieb.title)
        // }else return moviea.year - movieb.year;
    });
    return newMovies;    
}    
```
More about localeCompare: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare